src/crypto/libcrypto.c

```diff
@@ -170,6 +179,30 @@ _native_crypto_aes_256_cbc_decrypt (const _mongocrypt_buffer_t *key,
    return ret;
 }
 
+bool
+_native_crypto_aes_256_cbc_encrypt (const _mongocrypt_buffer_t *key,
+                                    const _mongocrypt_buffer_t *iv,
+                                    const _mongocrypt_buffer_t *in,
+                                    _mongocrypt_buffer_t *out,
+                                    uint32_t *bytes_written,
+                                    mongocrypt_status_t *status)
```

Comment:
This exact set of six parameters is repeated in many functions. Could they be encapsulated into a params structure to represent the parameters as named fields? That would be easier to pass around with accidentally transposing or confusing arguments.

Reply:
Added a `aes_256_args_t` type.